pkg/apis/serving/v1alpha2/framework_triton.go

```diff
@@ -41,7 +41,7 @@ func (t *TritonSpec) GetContainer(modelName string, parallelism int, config *Inf
 		Resources: t.Resources,
 		Args: []string{
 			"trtserver",
-			"--model-store=" + constants.DefaultModelLocalMountPath,
+			"--model-store=" + constants.DefaultModelLocalMountPath + "/" + modelName,
```

Comment:
This breaks triton as it always expects the model repository path, the model name is specified in metadata file pbtxt in each model subdir https://docs.nvidia.com/deeplearning/triton-inference-server/master-user-guide/docs/model_repository.html#repository-layout